partcad/src/partcad/part_factory_scad.py

```
@@ -86,3 +157,14 @@ async def instantiate(self, part):
             self.ctx.stats_parts_instantiated += 1
 
             return shape
+
+    async def prepare_python(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `prepare_python`

Add return type annotation: `None`

(ANN201)